apps/codeforafrica/src/components/ArticlePage/ArticlePage.js

```diff
@@ -31,13 +30,13 @@ function ArticlePage({
         alt={title}
         objectFit="cover"
         priority
-        src={image?.src}
+        src={featureImage}
       />
       <ArticleHeader
         title={title}
-        date={date}
+        date={publishedAt}
         tags={tags}
-        summary={summary}
+        summary={excerpt || customExcerpt}
```

Comment:
Lets also do this in lib... I think `customExcerpt` means hand-written excerpt, right? if yes, then it should take precedence over general `excerpt`.

Reply:
@kilemensi How do we do this in the lib? I'm not sure how to approach it.

Reply:
The same way/place you do the camelCase conversion @kelvinkipruto . e.g. you can add something link :point_down: [here](https://github.com/CodeForAfrica/ui/blob/72d551ed4301c58f045a960023413f9073a83987/apps/codeforafrica/src/lib/api.ghost.js#L18-L25).
```js
  const posts = await api.posts.browse({
    limit,
    page,
    fields: "id,title,slug,published_at,feature_image,excerpt",
    include: "authors,tags",
    ...other,
  });
  return posts.map((post) => {
    const { customExcerpt, excerpt: originalExcerpt, ...other } = convertToCamelCase(post);
    const excerpt = customExcerpt || originalExcerpt || null;
    // Any other manipulation can also be done here
    return { ...other, excerpt }; 
  });
```

Reply:
Ok, so instead of returning both customExcerpt and excerpt, we just return one of them?

Reply:
Exactamundo! We only need one.